src/microsoft/office/license/user/order/microsoft-office-license-user-order.html

```diff
@@ -0,0 +1,62 @@
+<div data-ng-controller="MicrosoftOfficeLicenseUserOrderCtrl as orderUserCtrl">
+    <div data-wizard
+         data-wizard-bread-crumb
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="orderUser"
+         data-wizard-title="i18n.microsoft_office_license_user_order">
+
+        <div data-wizard-step
+             data-wizard-step-valid="orderUserCtrl.number && orderUserCtrl.license">
+
+            <form class="form-horizontal mt-4" name="microsoftOfficeLicenseOrderUserForm">
+                <div class="form-group"
+                     data-ng-class="{'has-error': microsoftOfficeLicenseOrderUserForm.number.$dirty && microsoftOfficeLicenseOrderUserForm.number.$invalid}">
+                    <label class="control-label col-md-4 required" for="number"
+                           data-i18n-static="microsoft_office_license_order_user_number_label"></label>
+                    <div class="col-md-6">
+                        <input type="number" class="form-control" id="number" name="number" min="0" required
```

Comment:
`min` should be backed up by`data-ng-min` as `min` is not supported by older browsers